lldb/source/Plugins/Platform/gdb-server/PlatformRemoteGDBServer.cpp

```diff
@@ -684,6 +684,14 @@ Status PlatformRemoteGDBServer::RunShellCommand(
                                           signo_ptr, command_output, timeout);
 }
 
+bool PlatformRemoteGDBServer::CalculateMD5(const FileSpec &file_spec,
+                                           uint64_t &low, uint64_t &high) {
+  if (!IsConnected()) {
+    return false;
+  }
```

Comment:
The LLVM coding standards suggest not having curly braces for 1 line bodies: https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements